docs/Config_Reference.md

```diff
@@ -1817,6 +1817,10 @@ z_offset:
 #   samples. If this tolerance is exceeded then either an error is
 #   reported or the attempt is restarted (see
 #   samples_tolerance_retries). The default is 0.100mm.
+#samples_discard_first: 0
```

Comment:
I am not sure what the status of this PR is but there are talks of resurrection.

Would 'samples_discard' be more appropriate?